Reneiski_Homework1/Reneiski_Homework1/Properties/launchSettings.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
I don't think you need this file and folder 'Properties' in general...

Reply:
Fixed

Reply:
Fixed